src/client/driver/command-executors/execute-action-command.js

```diff
@@ -148,6 +152,15 @@ function ensureCommandElements (command, timeout) {
         });
 }
 
+function ensureCommandArguments (command) {
+    if (command.type === COMMAND_TYPE.pressKey) {
+        var parsedKeySequence = parseKeySequence(command.keys);
```

Comment:
OK, since it require some additional investigation/work I suggest to keep it as is. Because we don't have time for this now.